lib/chai/utils/addProperty.js

```diff
@@ -40,8 +39,9 @@ module.exports = function (ctx, name, getter) {
 
   Object.defineProperty(ctx, name,
     { get: function addProperty() {
+        var keep_ssfi = flag(this, 'keep_ssfi');
```

Comment:
@Turbo87, sorry but I didn't understand what you just said :(

Tests for this should be similar to what we already have, we just need to overwrite the property and them check the stack trace to make sure it does not include that `_super` call. Basically you've got to ensure the second example you posted on [this issue](https://github.com/chaijs/chai/issues/659) won't happen.